bgpd/bgp_ecommunity.c

```diff
@@ -263,8 +263,11 @@ struct ecommunity *ecommunity_dup(struct ecommunity *ecom)
 }
 
 /* Return string representation of ecommunities attribute. */
-char *ecommunity_str(struct ecommunity *ecom)
+const char *ecommunity_str(struct ecommunity *ecom)
 {
+	if (!ecom)
+		return "(null)";
```

Comment:
Do we need same for other _str() functions (community_str, lcommunity_str)?

Reply:
I checked the other ones and they already have something returned in that case.